constellation-app/constellation
# Dragging Timeline summary box bug fix

### Description of the Change

Fixed the bug where the summary box in timeline view would shrink when trying to drag it past the right edge.

This was fixed by moving a line of code (an incorrect assumption in its current place) to more appropriate places in the code. 

### Alternate Designs



### Why Should This Be In Core?

Timeline view is in Core

### Benefits
Summary box no longer unexpectedly shrinks

### Possible Drawbacks



### Verification Process

1) Open a sphere graph
2) Open timeline view ensuring summary window is visible
3) Drag box to the right edge and observe it no longer shrinks and jumps around

### Applicable Issues

#30
